### ISSUE TITLE
Configuring Patterns - Error mapping_parsing_exception

### ISSUE THREAD

**Issue by binarywaddle** [pfelk/pfelk#78]
Hi, I'm very new to ELK and tried to give it a whirl. I'm using Elasticsearch 6.8.7 on an Ubuntu Server 18.04 VM. I'm trying to transmit logs from my OPNSense VM to my logging server with ELK stack. When I get to the Configuration portion and try to configure the patterns, I receive this error message when sending the request:

```
{
  "error": {
    "root_cause": [
      {
        "type": "mapper_parsing_exception",
        "reason": "Root mapping definition has unsupported parameters:  [destination : {properties={geo={dynamic=true, properties={ip={type=ip}, location={type=geo_point}}}, as={dynamic=true, properties={ip={type=ip}, location={type=geo_point}}}}}] [source : {properties={geo={dynamic=true, properties={ip={type=ip}, location={type=geo_point}}}, as={dynamic=true, properties={ip={type=ip}, location={type=geo_point}}}}}]"
      }
    ],
    "type": "mapper_parsing_exception",
    "reason": "Failed to parse mapping [properties]: Root mapping definition has unsupported parameters:  [destination : {properties={geo={dynamic=true, properties={ip={type=ip}, location={type=geo_point}}}, as={dynamic=true, properties={ip={type=ip}, location={type=geo_point}}}}}] [source : {properties={geo={dynamic=true, properties={ip={type=ip}, location={type=geo_point}}}, as={dynamic=true, properties={ip={type=ip}, location={type=geo_point}}}}}]",
    "caused_by": {
      "type": "mapper_parsing_exception",
      "reason": "Root mapping definition has unsupported parameters:  [destination : {properties={geo={dynamic=true, properties={ip={type=ip}, location={type=geo_point}}}, as={dynamic=true, properties={ip={type=ip}, location={type=geo_point}}}}}] [source : {properties={geo={dynamic=true, properties={ip={type=ip}, location={type=geo_point}}}, as={dynamic=true, properties={ip={type=ip}, location={type=geo_point}}}}}]"
    }
  },
  "status": 400
}
```
<img width="960" alt="kibana-pattern-configuration" src="https://user-images.githubusercontent.com/1279202/77205389-2632f300-6acb-11ea-8a88-d5543d3dfe02.png">

**Comment by revere521** [comment]
It could be a compatibility issue?  I think this project started with version 7+ of the ELK stack.  

is it possible to update to at least 7, or the current 7.6.2?

I've never seen that error before.

**Comment by a3ilson** [comment]
My initial take would be to upgrade Elasticsearch to v7.6.  However, it that is not practical give the following a try:

```
PUT _template/pf-location-template
{
  "index_patterns": [
    "pf-*"
  ],
  "mappings": {
    "type_name": {  
      "properties": {
        "destination": {
          "properties": {
            "geo": {
              "dynamic": true,
              "properties": {
                "ip": {
                  "type": "ip"
                },
                "location": {
                  "type": "geo_point"
                }
              }
            },
            "as": {
              "dynamic": true,
              "properties": {
                "ip": {
                  "type": "ip"
                },
                "location": {
                  "type": "geo_point"
                }
              }
            }
          }
        },
        "source": {
          "properties": {
            "geo": {
              "dynamic": true,
              "properties": {
                "ip": {
                  "type": "ip"
                },
                "location": {
                  "type": "geo_point"
                }
              }
            },
            "as": {
              "dynamic": true,
              "properties": {
                "ip": {
                  "type": "ip"
                },
                "location": {
                  "type": "geo_point"
                }
              }
            }
          }
        }
      }
    }
  }
}
```

Let me know if this works.

**Comment by binarywaddle** [comment]
Ahh, yes I see now. The guide I used to install the elkstack referred to the 6.x version. Upgraded to the latest available version (7.6.1) and that fixed it. Thank you guys!
